test/setup/deployers/operator-openshift.ts

```diff
@@ -8,10 +10,30 @@ export const operatorDeployer: IDeployer = {
 async function deployKubernetesMonitor(
   _imageOptions: IImageOptions,
 ): Promise<void> {
-  await kubectl.applyK8sYaml('./test/fixtures/operator/operator-source.yaml');
+  const overriddenOperatorSource = 'snyk-monitor-operator-source.yaml';
+  createTestOperatorSource(
+    overriddenOperatorSource,
+    process.env['QUAY_USERNAME']!,
```

Comment:
Noob Typescript question, but what is the `!` for here?

Reply:
A variable can be `type | undefined`, here the `!` operator "silences" the compiler and tells it "I am sure this variable is not undefined"

Reply:
TIL! Thanks. It was hard to google this :)

Reply:
Here's a better explanation 😄  https://stackoverflow.com/a/42274019